packages/experience-builder-sdk/src/blocks/VisualEditorBlock.tsx

```diff
@@ -40,18 +45,37 @@ type VisualEditorBlockProps = {
   areEntitiesFetched: boolean;
 };
 
+const designComponentTypes = ['DesignComponent', 'DesignComponentBlock'];
```

Comment:
- Constants should be in SCREAMING_SNAKE_CASE
- I would like to add `node` as `type` can mean a lot (e.g. content type, experience type)

```suggestion
const DESIGN_COMPONENT_NODE_TYPES = ['DesignComponent', 'DesignComponentBlock'];
```